admin-site/src/App.svelte

```diff
@@ -29,9 +31,15 @@
       method: 'GET'
     })
 
-    events = await response.json()
+    let grabbedFromServerEvents = await response.json()
     gotEvents = true
-    console.log(serverURL)
+    
+    storedEvents = writable(grabbedFromServerEvents);
+    storedEvents.subscribe(value => {
+      events = value;
+      console.log(value);
+    });
+
     focusedEvent = events[0]
   }
 
```

Comment:
When you start moving around logic, I think this cue broadcast function probably belongs in `Occasion`. The slider should emit an event (something like `activated`) that's consumed by `Occasion` and triggers cue broadcast.

Reply:
The pattern to keep in mind: Data Down, Actions Up. Parents provide data to children, children emit actions when Parents need to do something to update data. (Or a component handles everything internally)

Reply:
https://svelte.dev/tutorial/component-events